fedmsg/core.py

```diff
@@ -463,7 +467,7 @@ def _run_socket(self, sock, name, ep, watched_names=None):
                     else:
                         raise ValidationError(msg)
             else:
-                return name, ep, _topic, msg
+                return (name, ep, _topic, msg)
```

Comment:
Personally I see them as equivalent in explicitness, but it's not surprising to run across this and it doesn't hurt anything so I don't mind one way or the other.

As a side node, something I think _is_ more explicit and something I quite like in this sort of scenario is a namedtuple. However I think that's probably better as a separate PR.